src/content/language/collections.md

```diff
@@ -328,6 +332,68 @@ A map entry element has this syntax in a collection:
 <key_expression>: <value_expression>
 ```
 
+### Null-aware elements {: #null-aware-element }
+
+The null-aware element checks if a value is not null
+and inserts the resulting value into the surrounding
+collection. Null-aware elements can only be used with
+leaf elements and can't be nested.
```

Comment:
I'm not sure what "can only be used with leaf elements" is conveying here. You don't have it listed as a leaf element above, and other leaf elements obviously can't contain it (because they are leaf elements). Maybe just omit this sentence?

Reply:
I've deleted this sentence.